### PR TITLE
Document GitHub App usage, security model, etc.

### PR DESCRIPTION
**Pull request by nfagerlund**
At long last 😩

### CODE REVIEW

File: content/source/docs/cloud/vcs/github-app.html.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+[policy sets]: ../sentinel/manage-policies.html
+[vcs settings]: ../workspaces/vcs.html
+[create]: ../workspaces/creating.html
+[owners]: ../users-teams-organizations/teams.html#the-owners-team
+
+# Configuration-Free GitHub Usage
+
+These instructions are for using repositories from GitHub.com with Terraform Cloud workspaces, without configuring an OAuth connection.
+
+Using GitHub this way does not require any action from an [organization owner][owners]; anyone who can create workspaces or administer existing workspaces can connect GitHub repositories.
```
Comment (judithpatudith) [inline content/source/docs/cloud/vcs/github-app.html.md]:
```suggestion
Using GitHub this way does not require any action from a Terraform [organization owner][owners]; anyone who can create workspaces or administer existing workspaces can connect GitHub repositories.
```

File: content/source/docs/cloud/vcs/github-app.html.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+This method uses a preconfigured GitHub App, and only works with GitHub.com. There are separate instructions for connecting to [GitHub.com via OAuth](./github.html), connecting to [GitHub Enterprise](./github-enterprise.html), and connecting to [other supported VCS providers.](./index.html)
+
+## Using GitHub Repositories
+
+You can connect GitHub repositories to Terraform Cloud workspaces without changing any organization settings.
```
Comment (judithpatudith) [inline content/source/docs/cloud/vcs/github-app.html.md]: I think we can cut this line; the section above explains it.

File: content/source/docs/cloud/vcs/github-app.html.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+To do so, choose "GitHub.com" on the "Connect to a version control provider" screen, which is shown when [creating a new workspace][create] or [changing a workspace's VCS connection][vcs settings]. Authorize access to GitHub if necessary. On the next screen, select a GitHub account or organization from the drop-down menu (or add a new organization) and choose a repository from the list.
```
Comment (judithpatudith) [inline content/source/docs/cloud/vcs/github-app.html.md]:
```suggestion
Choose "GitHub.com" on the "Connect to a version control provider" screen, which is shown when [creating a new workspace][create] or [changing a workspace's VCS connection][vcs settings]. Authorize access to GitHub if necessary. On the next screen, select a GitHub account or organization from the drop-down menu (or add a new organization) and choose a repository from the list.
```

File: content/source/docs/cloud/vcs/github-app.html.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+The controls on the "Connect to a version control provider" screen can vary, depending on your permissions and your organization's settings:
+
+- In organizations with no VCS connections configured:
+    - [Owners][] will see several drop-down menus, sorted by product family. Choose "GitHub.com" (_not_ "GitHub.com (Custom)") from the GitHub menu.
+    - Non-owners will see a "GitHub" button.
+- In organizations with an existing VCS connection, only the connected providers are shown. Click the "Connect to a different VCS" link to reveal the provider menus (owners) or the GitHub button (non-owners).
+
+![Screenshot: the "Connect to a version control provider" screen of the "Create a new workspace" page.](./images/gh-app-choose-provider.png)
+
+![Screenshot: Terraform Cloud's repository list, with the organization switcher menu open and the "Add another organization" option visible.](./images/gh-app-install.png)
```
Comment (judithpatudith) [inline content/source/docs/cloud/vcs/github-app.html.md]: Move the screen shots next to the bullet that describes them.

File: content/source/docs/cloud/vcs/github-app.html.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+## GitHub Permissions
+
+Unlike OAuth connections, the Terraform Cloud GitHub App does not configure a Terraform Cloud organization to always act as one particular GitHub user. Instead, each Terraform Cloud user authenticates individually, and can use GitHub resources within Terraform Cloud according to their own GitHub organization memberships and access permissions.
```
Comment (judithpatudith) [inline content/source/docs/cloud/vcs/github-app.html.md]: There are a ton of negatives in this first sentence and I found it a bit hard to follow. Can we phrase it positively instead, or put the second sentence before the first?

File: content/source/docs/cloud/vcs/github-app.html.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+To enable this, Terraform Cloud requests two kinds of permissions:
+
+- **Per user:** Each Terraform Cloud user must _authorize_ Terraform Cloud for their own GitHub account. This lets Terraform Cloud determine which organizations and repositories you have access to.
```
Comment (judithpatudith) [inline content/source/docs/cloud/vcs/github-app.html.md]:
```suggestion
- **Per user:** Each Terraform Cloud user must _authorize_ Terraform Cloud for their own GitHub account. This lets Terraform Cloud determine which organizations and repositories they have access to.
```

File: content/source/docs/cloud/vcs/github-app.html.md (hunk 3 above)
Comment (nfagerlund) [reply content/source/docs/cloud/vcs/github-app.html.md]: 👍🏼

File: content/source/docs/cloud/vcs/github-app.html.md (hunk 4 above)
Comment (nfagerlund) [reply content/source/docs/cloud/vcs/github-app.html.md]: These don't correspond to bullets, they correspond to sentences in the first paragraph in this section. (Er, first after removing the initial line, per above.) 

I debated putting them between the primary paragraph and the trailing explanation of how those controls vary, but that pushed the explanation way down the page, where it seemed lonesome. 

??

File: content/source/docs/cloud/vcs/github-app.html.md (hunk 5 above)
Comment (nfagerlund) [reply content/source/docs/cloud/vcs/github-app.html.md]: Hmm, you're right. I'll do both, probably.

Comment (nfagerlund) [reply content/source/docs/cloud/vcs/github-app.html.md]: Shrinking that first sentence and moving it down into a "-> note".

**Comment by nfagerlund** [comment]
Three ✅ s seems good to me, so I'm going to merge! @judithpatudith, I'm leaving the screenshots where they are for now, per my reply; if you want them elsewhere, I can do a follow-up PR later. 👍🏼
